generators/app/index.js

```diff
@@ -95,6 +97,34 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      var done = this.async();
+
+      githubUsername(this.user.git.email(), function (err, username) {
+        if (err) {
+          this.log.error(err);
```

Comment:
Yes, either we handle the exception or we throw. I'm fine if we handle it correctly.